core/controllers/profile_test.py

```diff
@@ -29,233 +29,358 @@
 import utils
 
 
-class SignupTests(test_utils.GenericTestBase):
+class ProfilePageTests(test_utils.GenericTestBase):
```

Comment:
I'm finding it hard to review this file -- see https://github.com/oppia/oppia/pull/8022/files. What exactly changed?

If it's a reordering, perhaps consider doing that part in a separate PR? Otherwise the diff gets all messed up...

Reply:
Yes, I reorganized a bit, it should be enough to review https://github.com/oppia/oppia/blob/c569baf1c168071173bccd7bd0993763478c0973/core/controllers/profile_test.py#L776 class and https://github.com/oppia/oppia/blob/c569baf1c168071173bccd7bd0993763478c0973/core/controllers/profile_test.py#L765 class. The other changes are just moving stuff around.